ironfish-cli/src/trusted-setup/server.ts

```diff
@@ -170,7 +170,7 @@ export class CeremonyServer {
   private onConnection(socket: net.Socket): void {
     const client = new CeremonyServerClient({ socket, id: uuid(), logger: this.logger })
     this.queue.push(client)
-    client.send({ method: 'joined', queueLocation: this.queue.length })
+    client.send({ method: 'joined', queueLocation: this.queue.length - 1 })
```

Comment:
Is this right? if there are 4 people in the queue (including them) then their position would be 4 right?